src/app/shared-component/wizard/wizard-steps.ts

```diff
@@ -0,0 +1,9 @@
+import { WizardStep } from './wizard-step';
+
+export interface WizardSteps {
```

Comment:
This part looks to specific to be in ngx-widgets. If we want to move the other parts of "wizard" this will either need to stay or be changed. 
Or we keep all of it here but then is it really a shared component?